scripts/check-test-files.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -euo pipefail
+
+DIR="$(cd "$(dirname "${0}")/.." && pwd)"
+source "${DIR}/scripts/source.sh"
+cd "${DIR}"
+
+
+NO_TEST_FILE_DIRS=""
+for dir in $(not_cover_ignore_dirs); do
+  if [ -z "$(find "${dir}" -name '*_test\.go')" ]; then
+    if [ -z "${NO_TEST_FILE_DIRS}" ]; then
+      NO_TEST_FILE_DIRS="${dir}"
+    else
+      NO_TEST_FILE_DIRS="${NO_TEST_FILE_DIRS} ${DIR}"
+    fi
+  fi
+done
+
+if [ -n "${NO_TEST_FILE_DIRS}" ]; then
+  echo "error: at least one _test.go file must be in these directories: ${NO_TEST_FILE_DIRS}" >&2
```

Comment:
Creating an empty test file is more difficult - you have to detect the package name. I can do this, but do you think this is necessary? I rather do this in a separate diff.